workflows/dnaseq/dnaseq-core.wdl

```diff
@@ -0,0 +1,109 @@
+## **WARNING:** this workflow is experimental! Use at your own risk!
+
+version 1.1
+
+import "../../data_structures/read_group.wdl"
+import "../../tools/bwa.wdl"
+import "../../tools/picard.wdl"
+import "../../tools/samtools.wdl"
+import "../../tools/util.wdl"
+import "../general/bam-to-fastqs.wdl" as bam_to_fastqs_wf
+import "../general/samtools-merge.wdl" as samtools_merge_wf
+
+workflow dnaseq_core_experimental {
+    meta {
+        description: "Aligns DNA reads using bwa"
+        outputs: {
```

Comment:
Asking the question to be thorough: Should there be an `intermediate_files` struct to match our other workflows?

I think not. This workflow's intermediate files are chunked, which makes them less useful and harder to manage than the intermediate files of our other workflows. I could be persuaded otherwise, but I'm going out on a limb by assuming you agree it's not worth the hassle.

Reply:
I agree. I don't think the chunked files are useful except as true intermediates in this workflow.